src/Microsoft.Azure.WebJobs.ServiceBus/EventHubs/IEventHubProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    // Expose to binders so they can get the EventHub connections. 
+    interface IEventHubProvider
```

Comment:
If this is an interface users implement, you should consider making this an abstract class instead so we can add to it w/o that being a breaking change. That's the approach I've taken with all the other binding config extensibility points.